src/convertToObject.js

```diff
@@ -7,6 +7,15 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((arrKeyAndValue) => arrKeyAndValue.split(':'))
+    .filter((item) => item.length > 1)
+    .reduce((objectRes, arrTwoWords) => {
```

Comment:
The variable name 'objectRes' is not descriptive and does not conform to the [CODE STYLE] guideline to avoid namings like 'array', 'object', 'arr', 'obj', 'result'. A more descriptive name, such as 'styleObject', would improve code readability.